requirements.txt

```diff
@@ -22,3 +22,4 @@ SQLAlchemy==1.4.41
 cryptography>=41.0.0
 python-dateutil==2.8.2
 slack-sdk==3.20.2
+tabulate==0.9.0
```

Comment:
These are the server requirements, can you add tabulate to the `setup.py` requirements instead?

Reply:
Cheers, removed from requirements.txt and added to setup.py instead